content/join/policy.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Policy
+parent: Joining the infrastructure
+nav_order: 10
+---
+
+# Policy
+
+In order to ensure proper testbed operation and maintenance we require that all sites (external and internal) abide by the policies outlined below.
+
+1. Provide one or more dedicated machines on-site to act as the SCIONLab nodes. The machine should not be used for any other tasks beyond its role as an SCIONLab node.
+2. Provide root (or sudo) access to the SCIONLab management institution (currently ETH Zurich).
+3. Allow the SCIONLab management institution to install, remove and update SCIONLab related software as needed for the operation and maintenance of the network.
+4. Work with the SCIONLab management institution to solve any firewall and tunneling issues that may arise during the installation, configuration and operation of the SCIONLab node(s).
+5. Provide the name, email and telephone information of an easily and quickly reachable site-specific operator who will be the main contact whenever issues arise with the local SCIONLab node(s). A backup operator, while not required, is highly encouraged. Operator email addresses must belong to the institution. All local operators will be required to join and regularly monitor an SCIONLab community mailing list and are required to respond to requests from the testbed management institution in a reasonable time, but no more than 3 working days.
```

Comment:
English nitpick: s/an SCIONLab/a SCIONLab/
(Or maybe actually the, not a, because we mean a specific one? Do we?)